src/components/UserList.js

```diff
@@ -1,23 +1,75 @@
 import React from 'react';
 import UserItem from './UserItem';
+import {connect} from 'react-redux';
+import {addUserAction} from '../actions/actions';
+
+const mapActionToProps = {
+	userAdd: addUserAction
+};
+const mapStateToProps = (state, props) => {
+	return {
+		users: state.users
+	};
+};
```

Comment:
Często ten element występuje poniżej deklaracji komponentu (klasy) ponieważ to "tylko" szczegół implementacyjny.
Zazwyczaj jak chcemy zobaczyć co robi komponent to interesuje nas jego zawartość, a nie implementacja reduxa.